eclass/intel-sdp-r1.eclass

```diff
@@ -284,10 +304,11 @@ _isdp_version_test() {
 _isdp_run-test() {
 	debug-print-function ${FUNCNAME} "${@}"
 
-	if [[ -z ${INTEL_SKIP_LICENSE} ]]; then
+	if [[ -z "${INTEL_SKIP_LICENSE}" ]]; then
```

Comment:
No need for quoting. Not that I haveanything against it, just wondering why you added it.